src/components/ApprovalWorkflowSection.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import {View} from 'react-native';
+import useLocalize from '@hooks/useLocalize';
+import useTheme from '@hooks/useTheme';
+import useThemeStyles from '@hooks/useThemeStyles';
+import type ApprovalWorkflow from '@src/types/onyx/ApprovalWorkflow';
+import Icon from './Icon';
+import * as Expensicons from './Icon/Expensicons';
+import MenuItem from './MenuItem';
+import PressableWithFeedback from './Pressable/PressableWithFeedback';
+import Text from './Text';
+
+function ApprovalWorkflowSection({approvalWorkflow}: {approvalWorkflow: ApprovalWorkflow}) {
+    const styles = useThemeStyles();
+    const theme = useTheme();
+    const {translate} = useLocalize();
+
+    return (
+        <PressableWithFeedback
+            accessibilityRole="button"
+            style={[styles.border, styles.p4, styles.flexRow, styles.justifyContentBetween, styles.mt6, styles.mbn3]}
+            onPress={() => {}}
+            accessibilityLabel={translate('workflowsPage.addApprovalsTitle')}
+        >
+            <View style={[styles.flex1]}>
+                {approvalWorkflow.isDefault && (
+                    <View style={[styles.flexRow, styles.mb4, styles.alignItemsCenter, styles.pb1, styles.pt1]}>
+                        <Icon
+                            src={Expensicons.Lightbulb}
+                            fill={theme.icon}
+                            additionalStyles={styles.mr2}
+                            small
+                        />
+                        <Text
+                            style={[styles.textLabelSupportingNormal]}
+                            suppressHighlighting
+                        >
+                            {translate('workflowsPage.addApprovalTip')}
+                        </Text>
+                    </View>
+                )}
+                <MenuItem
+                    title={translate('workflowsExpensesFromPage.title')}
+                    style={styles.p0}
+                    titleStyle={styles.textLabelSupportingNormal}
+                    descriptionTextStyle={styles.textNormalThemeText}
+                    description={approvalWorkflow.isDefault ? translate('workspace.common.everyone') : approvalWorkflow.members.map((m) => m.displayName).join(', ')}
+                    icon={Expensicons.Users}
+                    iconHeight={20}
+                    iconWidth={20}
+                    iconFill={theme.icon}
+                    interactive={false}
+                />
+
+                {approvalWorkflow.approvers.map((approver) => (
+                    <View key={approver.email}>
+                        <View style={{height: 16, width: 1, backgroundColor: theme.border, marginLeft: 19}} />
+                        <MenuItem
+                            title={translate('workflowsPage.approver')}
```

Comment:
In case there are more approvers, you need to add ordinals numbers:

![image](https://github.com/user-attachments/assets/f47b62ce-75c4-4272-b315-1e0763cafd8f)